src/Shell/Task/TemplateTask.php

```diff
@@ -431,6 +446,8 @@ public function getOptionParser()
             'help' => 'The controller name if you have a controller that does not follow conventions.'
         ])->addOption('prefix', [
             'help' => 'The routing prefix to generate views for.',
+        ])->addOption('index-columns', [
+            'help' => 'Limit for the number of index columns (Default: 7)'
```

Comment:
You can set the default in the option definition. Then the help formatter will output the default for you.